README.md

```diff
@@ -1,18 +1,15 @@
-# Positive Work Environment Task Force
-
+# Positive Work Environment Community Group
 
 W3C is a global community where participants choose to work together. In that community, we experience differences in language, location, nationality, and experience. In such a diverse environment, misunderstandings and disagreements happen, and in most cases can be resolved informally.
 
 This is the GitHub repository for the [Positive Work Environment Community Group](https://www.w3.org/community/pwe/).
 
 The CG picks up the work of the former [PWE Task Force](https://www.w3.org/Consortium/pwe/). 
 
-Chairs: Tzviya Siegman, Angel Li
+Chairs: Tzviya Siegman, Wendy Reid, Liz Lutgendorff
 
-The document can also be read directly on the [github.io page](https://w3c.github.io/PWETF/)
+This group maintains the W3C Code of Ethics and Professional Conduct, the draft version of the document can be read directly on the [github.io page](https://w3c.github.io/PWETF/)
```

Comment:
```suggestion
This group maintains the [W3C Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/).
The latest editor's draft of the document can be read directly on the [github.io page](https://w3c.github.io/PWETF/).
```

My preference would be not to use "can be read directly on the github.io page." but to show the URL directly, for example "is at https://w3c.github.io/PWETF/ "

Reply:
Sorry @wareid , I probably wasn't very clear. My suggested change was to link directly to the active CEPC, and I also wanted less formally to suggest a change to the wording for the link to the editor's draft. Only the latter change was applied.

I've just realised something else: it is probably worth a brief mention of the eventual owner of the CEPC - I suspect that publishing a new version is not the decision of the PWE CG, but of a more formally constituted group, and it is probably worth briefly summarising that so that folk understand that merely having a change made in the Editor's Draft doesn't necessarily make it happen in the official version.